test/libsolidity/util/TestFileParser.cpp

```diff
@@ -271,6 +273,19 @@ FunctionCallArgs TestFileParser::parseFunctionCallArguments()
 	return arguments;
 }
 
+vector<string> TestFileParser::parseFunctionCallReactions()
+{
+	vector<string> result;
+	while (accept(Token::Reaction, false))
```

Comment:
Shouldn't this parse `-` instead of the `reaction` keyword?

Reply:
Thats true, I was not sure how this could be done nicely.

Reply:
I don't understand - can't you use `-` (or it's token name) instead of `Token::Reaction`?